www/src/components/shared/pullquote.js

```diff
@@ -1,194 +1,165 @@
-  text-indent: ${p => p.theme.space[7]};
-  margin: ${p => p.theme.space[8]} 0;
-
-  /* needed for overriding typography.js style "p *:last-child {"" */
-  p > & {
-    margin: ${p => p.theme.space[8]} 0;
-  }
-
-  ${mediaQueries.lg} {
-    line-height: ${p => p.theme.lineHeights.loose};
-    margin: ${p => p.theme.space[8]} ${p => (p.narrow ? 0 : `-3.5rem`)};
-    padding: 2.8rem 3.5rem;
-    text-indent: 1.8rem;
-
-    p > & {
-      margin: ${p => p.theme.space[8]} ${p => (p.narrow ? 0 : `-3.5rem`)};
-    }
-  }
-`
-
-const Citation = styled(`cite`)`
-  display: block;
-  font-style: italic;
-  font-weight: normal;
-  margin-top: ${p => p.theme.space[4]};
-  text-align: right;
-`
-
-const QuotationMark = styled(`span`)`
-  display: flex;
-  left: ${p => p.theme.space[8]};
-  position: absolute;
-  top: ${p => p.theme.space[7]};
-
-  svg {
-    fill: ${p => p.theme.colors.purple[80]};
-  }
 
-  ${mediaQueries.lg} {
-    left: 3rem;
-    top: 2.8rem;
-
-    svg {
-      fill: ${p => p.theme.colors.purple[80]};
-      transform: scale(1.1);
-    }
-  }
-`
-
-const Star = styled(`span`)`
-  display: flex;
-  position: absolute;
-
-  svg {
-    height: 100%;
-    width: 100%;
-  }
-
-  :nth-of-type(1) {
-    height: 20px;
-    left: 0;
-    top: 1.8rem;
-    transform: translateX(-50%);
-    width: 20px;
-
-    svg {
-      fill: ${p => p.theme.colors.yellow[40]};
-    }
-
-    ${mediaQueries.lg} {
-      height: 27px;
-      width: 27px;
-    }
-
-    .variantB & {
-      left: auto;
-      right: 0;
-      top: ${p => p.theme.space[7]};
-      transform: translate(50%, 0);
-    }
+function Citation({ author }) {
+  return (
+    <cite
+      sx={{
+        display: `block`,
+        fontStyle: `italic`,
+        fontWeight: `normal`,
+        mt: 4,
+        textAlign: `right`,
+      }}
+    >
+      &mdash; {author}
+    </cite>
+  )
+}
 
-    .variantC & {
-      bottom: 0;
-      left: auto;
-      right: 12rem;
-      top: auto;
-      transform: translate(0, 50%);
-    }
-  }
+function QuotationMark() {
+  return (
+    <span
+      sx={{
+        display: `flex`,
+        position: `absolute`,
+        left: [8, null, null, 9],
+        top: [7, null, null, `2.8rem`],
+      }}
+    >
+      <QuotationMarkOrnament
+        sx={{
+          fill: `purple.80`,
+          transform: [null, null, null, `scale(1.1)`],
+        }}
+      />
+    </span>
+  )
+}
 
-  :nth-of-type(2) {
-    left: 5rem;
-    height: 14px;
-    top: 0;
-    transform: translateY(-50%);
-    width: 14px;
+const starStyle = {
+  yellow: { size: [`20px`, null, null, `27px`], fill: `yellow.40` },
+  teal: { size: `14px`, fill: `teal.40` },
+  red: { size: `12px`, fill: `red.60` },
+}
 
-    svg {
-      fill: ${p => p.theme.colors.teal[40]};
-    }
-    .variantB & {
-      bottom: 0;
-      left: auto;
-      right: 3rem;
-      top: auto;
-      transform: translate(0, 50%);
-    }
+const starPositionStyles = [
+  {
+    yellow: { side: `left`, offset: `1.8rem` },
```

Comment:
@fk another thing we can do is switch these to percentage points.

So for this one, if `offset: `50%`, it will always be 50% between the top and bottom regardless of how tall the pullquote is. What do you think?

Reply:
If you think it still works out visually, go for it!